src/main/java/com/spotify/futures/FuturesExtra.java

```diff
@@ -130,7 +129,7 @@ public void onFailure(Throwable t) {
    * @throws NullPointerException if the {@param futures} is null
    */
   public static <T> ListenableFuture<T> select(final List<ListenableFuture<T>> futures) {
-    requireNonNull(futures);
+    Preconditions.checkNotNull(futures);
```

Comment:
True, but in case something gets refactored later it might be nice to be explicit about it.